content/departments/product-engineering/engineering/enablement/dev-experience/index.md

```diff
@@ -88,11 +89,20 @@ The goal is to have someone lead on identifying the right person to drive a fix
 The on-call support teammate should monitor the pipeline through channels like #buildkite-main for [flakes](https://docs.sourcegraph.com/dev/background-information/testing_principles#flaky-tests) and notifications from [`buildchecker`](https://docs.sourcegraph.com/dev/background-information/continuous_integration#buildchecker).
 If there are any issues, ensure issues are followed up on:
 
-1. Infer the owner based on the contents of the issue, e.g. through product names and other context, and reach out for assistance:
+1. Infer the owner based on the contents of the issue \*e.g. through product names, git history, and/or other context) and reach out for assistance:
    1. If a team can be inferred, ping the `@$TEAM-support` handle in Slack for assistance, escalating to `@$TEAM` if no support handle or teammate is available.
    2. If no team is easily inferred, ping the most recent author via `git blame` where relevant for assistance.
 2. Guide the teammate towards a resolution for the issue by following our [broken builds process](https://docs.sourcegraph.com/dev/background-information/testing_principles#broken-builds-on-the-main-branch) (also see [Continuous integration: Flakes](https://docs.sourcegraph.com/dev/background-information/continuous_integration#flakes)).
 
+##### CI support responsibilities
+
+The DevX team is _not_ responsible for all tools and tests that _run_ in Sourcegraph's CI infrastructure.
```

Comment:
It seems clear (and reasonable) to me that DevX is not responsible for fixing other teams' failing or unstable tests, but our responsibilities around tooling still feel a little fuzzy to me. Is it that we make decisions about the CI tools to use, best practices, and processes, but aren't responsible for monitoring the behavior of those tools?

Reply:
@taylorsperry my intention here is to express a _non-ownership by default_ model, because more often that not the inclination appears to be _devx ownership by default_. So this statement starts with a catch-all statement of non-responsibility, and then points to the section that does outline our responsibilities

Reply:
> Is it that we make decisions about the CI tools to use, best practices, and processes, but aren't responsible for monitoring the behavior of those tools?

We don't make decisions on the vast majority of CI tools, such as puppeteer etc, and I don't think we are the one-and-only governing body of standards either - other engineering teams bring forwards their own standards on a frequent basis. We can do the same _as an engineering team_ but not as _the_ devx team, if that makes sense, in the spirit of [our guiding principles](https://handbook.sourcegraph.com/strategy-goals/strategy/enablement/dev-experience#guiding-principles):

> We don’t own the developer experience at Sourcegraph – we simply focus on it. Sourcegraph engineers own the developer experience as a collective.

Reply:
I've tried to tweak the wording a bit in 4837e4562fe55b113e563075efc53f55b35abf4b - let me know what you think!